### PR TITLE
Update game over message

### PR DESCRIPTION
**Pull request by shrubsies**
reviewer setup

### CODE REVIEW

**Comment by RB1234567890-star** [comment]
Looks good.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (RB1234567890-star) [inline index.html:50]: Suggestion.

Comment (RB1234567890-star) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again!</h2>
```

Comment (shrubsies) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again !</h2>
```

**Pull request merged**